include/sir/config.h

```diff
@@ -93,12 +93,23 @@
 # define SIR_PIDSUFFIX ")"
 
 /**
- * The format for the current process/thread ID.
+ * The format for the current process ID.
  *
  * @remark Only applies if ::SIRO_NOPID or ::SIRO_NOTID are not set.
```

Comment:
If you split them then this comment is inaccurate now; should just be NOPID

Reply:
I'll fix